### PR TITLE
Add RotationTime, DoorOpenTime, DoorCloseTime and MoveTime to Lift.cs

### PR DESCRIPTION
**Pull request by ghost**
MoveTime is total time for moving (AnimationTime + RotationTime + DoorOpenTime + DoorCloseTime)

### CODE REVIEW

File: Exiled.API/Features/Lift.cs [hunk 1]
```diff
@@ -173,6 +173,38 @@ public float AnimationTime
             set => Base._animationTime = value;
         }
 
+        /// <summary>
+        /// Gets or sets the <see cref="RotationTime"/>.
+        /// </summary>
+        public float RotationTime
+        {
+            get => Base._rotationTime;
+            set => Base._rotationTime = value;
```
Comment (louis1706) [inline Exiled.API/Features/Lift.cs:182]: Required Testing if client don't get desync

File: Exiled.API/Features/Lift.cs [hunk 2]
```diff
@@ -173,6 +173,38 @@ public float AnimationTime
...
+        }
+
+        /// <summary>
+        /// Gets or sets the <see cref="DoorOpenTime"/>.
+        /// </summary>
+        public float DoorOpenTime
+        {
+            get => Base._doorOpenTime;
+            set => Base._doorOpenTime = value;
```
Comment (louis1706) [inline Exiled.API/Features/Lift.cs:191]: Required Testing if it's correctly do it with client

File: Exiled.API/Features/Lift.cs [hunk 3]
```diff
@@ -173,6 +173,38 @@ public float AnimationTime
...
+        }
+
+        /// <summary>
+        /// Gets or sets the <see cref="DoorCloseTime"/>.
+        /// </summary>
+        public float DoorCloseTime
+        {
+            get => Base._doorCloseTime;
+            set => Base._doorCloseTime = value;
```
Comment (louis1706) [inline Exiled.API/Features/Lift.cs:200]: Required Testing if it's correctly do it with client

**Review by louis1706 — CHANGES_REQUESTED** [review]
Requested test

**Comment by SiphoxR** [comment]
soooooooooooooooooooooooooooo I tested and changed code lol

**Review by louis1706 — APPROVED** [review]

**Pull request merged**